### PR TITLE
Added frontend pontsun

### PR DESCRIPTION
**Pull request by JonasNiestroj**

### CODE REVIEW

**Comment by schtibe** [comment]
I checked the changes, but it might be better if @thetractor does it too, I'm not much of a docker expert.

**Comment by 21adrian1996** [comment]
@thetractor I have merged the current master into this branch. The settings are now a bit more cleaned up.

So we are ready to merge.

File: docs/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM node:13.6.0-alpine3.11
+
+WORKDIR /code
```
Comment (thetractor) [inline docs/Dockerfile:3]: @21adrian1996 you are missing a new line here

File: frontend/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM node:13.6.0-alpine3.11
+
+WORKDIR /code
```
Comment (thetractor) [inline frontend/Dockerfile:3]: @21adrian1996 you are missing a new line here

File: frontend/package.json [hunk 3]
```diff
@@ -17,10 +17,13 @@
     "axios": "^0.19.2",
     "babel-polyfill": "^6.26.0",
     "core-js": "^3.4.4",
+    "npm": "^6.13.4",
```
Comment (thetractor) [inline frontend/package.json:20]: @21adrian1996 what kind of packages are these (`npm`, `run` and `serve`)? Never seen these in one of my Vue projects

File: docs/README.md [hunk 4]
```diff
@@ -1,16 +1,40 @@
...
+3. Run `INITIAL=1 docker-compose up` (if you use the fish shell you can run `set -x INITIAL 1; docker-compose up`)
+
+Note the `INITIAL` flag should not be set for subsequent container starts unless you want to reset the database.
 
-### Compiles and hot-reloads for development
-```
-npm start
-```
+After starting the containers you should have the following services avalable:
+- Django backend application: https://backend.docker.test
```
Comment (thetractor) [inline docs/README.md:22]: @21adrian1996 will this not be `http://api.forge.docker.test`?

File: frontend/package.json (hunk 3 above)
Comment (21adrian1996) [reply frontend/package.json:20]: @JonasNiestroj As this change was made by you: 
Can you explain why this is needed?

Comment (JonasNiestroj) [reply frontend/package.json:20]: They shouldnt be there. I think we can just remove them.

Comment (21adrian1996) [reply frontend/package.json:20]: @rotwurstesser Can you remove the packages?

After that, we can merge this merge request.

Comment (rotwurstesser) [reply frontend/package.json]: done, 🛳 ?

**Pull request merged**